Content.Server/GameObjects/Components/GUI/HandsComponent.cs

```diff
@@ -266,7 +281,7 @@ public bool Drop(IEntity entity, GridCoordinates coords, bool doMobChecks = true
                 throw new ArgumentException("Entity must be held in one of our hands.", nameof(entity));
             }
 
-            return Drop(slot, coords, doMobChecks);
+            return Drop(slot!, coords, doMobChecks);
```

Comment:
Here's one of the nullable warning suppressions.

```suggestion
            return Drop(slot, coords, doMobChecks);
```